CMakeLists.txt

```diff
@@ -884,3 +894,23 @@ if(BUILD_TESTS)
       --use-websockets
   )
 endif()
+
+# Generate and install CMake export file for consumers using CMake
+include(CMakePackageConfigHelpers)
+configure_package_config_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/cmake/ccf-config.cmake.in
+  ${CMAKE_BINARY_DIR}/cmake/ccf-config.cmake
+  INSTALL_DESTINATION
+  ${CMAKE_INSTALL_PREFIX}/cmake
+  PATH_VARS
+  CMAKE_INSTALL_LIBDIR
+  CMAKE_INSTALL_BINDIR
+  CMAKE_INSTALL_DATADIR
+  CMAKE_INSTALL_INCLUDEDIR)
```

Comment:
Do we need to pass all of these here?

Reply:
Not DATADIR, which I'm removing, but the others I believe we do, otherwise they won't expand correctly.